### PR TITLE
Make Completions Installable with Antibody

### PR DESCRIPTION
**Pull request by jGleitz**
Renames the zsh completions to start with an underscore and documents how to use [antibody](https://getantibody.github.io) to install them.

### CODE REVIEW

**Comment by jGleitz** [comment]
@googlebot I signed it!

**Comment by ahmetb** [comment]
I'm inclined to reject as since you can see the Readme is getting out of control wrt installation methods and I suspect Antibody isn't very popular. (Despite being a zsh user for years I heard it from you for the first time.)

**Comment by jGleitz** [comment]
> Antibody isn't very popular

My feeling is the opposite: When I got started with zsh, I felt that everybody had some plugin manager and that antibody was the most popular one. But then again, I have no data to back this up.

> the Readme is getting out of control

I agree. I see two options:

 1. Use subheadings instead of only bullet points. That would make it easier to parse what is happening where.
 2. Don’t document how to install with antibody (users knowing antibody well will figure it out themselves).


My main interest is to have the scripts renamed (https://github.com/ahmetb/kubectx/issues/262) so that installing automatically via antibody is possible. I wanted to document the process for fellow antibody users, but I certainly don’t insist on that.

**Comment by unixorn** [comment]
I agree with @ahmetb - anyone using a framework should know how to add a plugin. Also, if upstream changes how to install a plugin, then the information here becomes obsolete. That said, I also agree that prefixing the names with an `_` makes the completions more usable.

**Comment by jGleitz** [comment]
How can we proceed? Should I remove the README changes from this PR?

>  I think it might require some changes on the homebrew formula as well.

(from #262): where would these changes need to happen?

**Comment by ahmetb** [comment]
> (from #262): where would these changes need to happen?

Homebrew core repository "kubectx.rb" file.

**Comment by ahmetb** [comment]
I think we can merge this. 🤞  It might still break some downstream (alpine, debian pkgs etc) but it should be manageable.

**Comment by jGleitz** [comment]
@ahmetb glad that you will accept the PR :partying_face:!

To verify: Shall I remove the README changes before you merge?

**Comment by ahmetb** [comment]
readme is just getting really complicated. Is there a way to implement this and keep it just as simple without adding new paragraphs?

**Comment by jGleitz** [comment]
@ahmetb I have reworked the whole installation section of the `README`. From my point of view, it is now a lot easier to navigate and understand. I have linked the different sections from the overview at the beginning, I reduced the nesting, I moved the long bullet-point list into subsections and I reduced redundancies.

Does that work for you?

File: README.md [hunk 1]
```diff
@@ -97,22 +97,23 @@ Active namespace is "default".
...
 ### Kubectl Plugins (macOS and Linux)
 
-You can install and use [Krew](https://github.com/kubernetes-sigs/krew/) kubectl
-plugin manager to get `kubectx` and `kubens`. **NOTE:** This will not install
-shell completion scripts, if you want those, choose another installation method
-below.
+You can install and use the [Krew](https://github.com/kubernetes-sigs/krew/) kubectl
+plugin manager to get `kubectx` and `kubens`.
+
+ **NOTE:** This will not install the shell completion scripts. If you want them, choose another installation method
```
Comment (unixorn) [inline README.md:115]: There's an extra space before the format markers `**Note**`

File: README.md [hunk 2]
```diff
@@ -97,22 +97,23 @@ Active namespace is "default".
...
+or install the scripts [manually](#manual-installation-macos-and-linux)
```
Comment (unixorn) [inline README.md:116]: Please add a `.` after the link

**Review by unixorn — COMMENTED** [review]
Found a couple very minor formatting changes.

File: README.md (hunk 1 above)
Comment (jGleitz) [reply README.md]: Done.

File: README.md (hunk 2 above)
Comment (jGleitz) [reply README.md]: Done.

**Comment by jGleitz** [comment]
@unixorn Thank you! I fixed those.

**Comment by jGleitz** [comment]
@unixcorn @ahmetb can we finally merge this? I’d still be interested in the changes.

**Comment by jGleitz** [comment]
@ahmetb I fixed the merge conflict. I am looking forward to seeing this merged!

**Pull request merged**